nextclade/config/config_dict.yaml

```diff
@@ -0,0 +1,92 @@
+clade_repo: "https://raw.githubusercontent.com/influenza-clade-nomenclature/"
+
+builds:
+  h1n1pdm:
+    ha:
+      clade_systems:
+        clade:
+          url: "seasonal_A-H1N1pdm_HA/main/.auto-generated/clades.tsv"
+        subclade:
+          url: "seasonal_A-H1N1pdm_HA/main/.auto-generated/subclades.tsv"
+          key: "subclade"
+      refs:
+        CY121680:
+          filter: "--min-date 2014 --probabilistic-sampling  --exclude-where country='south_korea' --group-by year --min-length 1500  --subsample-max-sequences 1500"
```

Comment:
Why is South Korea excluded?

Reply:
I added a comment on this. There is a batch of south korean sequences with messed 5' UTR .